recipes/s2let/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1)
```

```suggestion
cmake_minimum_required(VERSION 3.8)
```

`c_std_99` has been added in CMake 3.8: https://cmake.org/cmake/help/v3.8/prop_gbl/CMAKE_C_KNOWN_FEATURES.html